### PR TITLE
Worfklow update

### PR DESCRIPTION
**Pull request by awright18**
Update github actions for ci and release

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -2,8 +2,8 @@ name: Release
 on:
   push:
     tags:
-      - '[6-9].[0-9]+.[0-9]+'
-      - '[6-9].[0-9]+.[0-9]+-*'
+      - '[6-9]+.[0-9]+.[0-9]+'
+      - '[6-9]+.[0-9]+.[0-9]+-*'
```
Comment (DavidBoike) [inline .github/workflows/release.yml:6]:
```suggestion
      - '[0-9]+.[0-9]+.[0-9]+'
      - '[0-9]+.[0-9]+.[0-9]+-*'
```

File: .github/workflows/release.yml [hunk 2]
```diff
@@ -14,84 +14,27 @@ jobs:
         uses: actions/checkout@v3.0.2
```
Comment (DavidBoike) [inline .github/workflows/release.yml:14]: Can't put a suggestion on the line but the runs-on should be ubuntu-20.04

File: .github/workflows/release.yml [hunk 3]
```diff
@@ -14,84 +14,27 @@ jobs:
...
+      - name: Deploy
+        uses: Particular/push-octopus-package-action@v1.0.0
         with:
-          server: https://deploy.particular.net
-          api_key: ${{ secrets.OCTOPUS_DEPLOY_API_KEY }}
-          project: ${{env.PARTICULAR_REPO_NAME}}
-          release_number: ${{env.MinVerVersion}}
-          package_version: ${{env.MinVerVersion}}
-          package: "GitReleaseManager:0.11.0"
+			    octopus-deploy-api-key: ${{ secrets.OCTOPUS_DEPLOY_API_KEY }}
```
Comment (DavidBoike) [inline .github/workflows/release.yml:40]: Are these tabs when it's expecting spaces? Why is that red?

**Review by DavidBoike — APPROVED** [review]

File: .github/workflows/ci.yml [hunk 4]
```diff
@@ -3,10 +3,7 @@ on:
   push:
     branches:
       - master
-      - release-[6-9].*
-  pull_request:
```
Comment (DavidBoike) [inline .github/workflows/ci.yml:7]: This is why CI didn't run. You removed the pull request trigger!

Comment (awright18) [reply .github/workflows/ci.yml:7]: Ah

**Pull request merged**